integrationtests/handshake/rtt.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("Handshake integration tets", func() {
 		expectedDuration := time.Duration(num) * rtt
 		Expect(testDuration).To(SatisfyAll(
 			BeNumerically(">=", expectedDuration),
-			BeNumerically("<", expectedDuration+rtt/3),
+			BeNumerically("<", expectedDuration+rtt/2),
```

Comment:
Probably. Let's see how this works out on our CIs first, I'd prefer to keep stricter constraints, if possible.